src/components/DoctorsPage/DoctorsCard/Schedule/Shedule.tsx

```diff
@@ -1,21 +1,63 @@
-import React from 'react';
+import React, { useState } from 'react';
 import Button from '@components/common/Button/Button';
 import SheduleItem from './SheduleItem/SheduleItem';
 import { ScheduleWrapper, WrapperHeader, ItemWrapper } from './SheduleStyle';
 
-const Shedule: React.FC = () => {
+interface IScheduleProps {
+  workTimeData: { date: string; time: string }[];
+}
+
+const Shedule: React.FC<IScheduleProps> = ({ workTimeData }) => {
+  const [choiceWorkTime, setChoiceWorkTime] = useState({
+    date: '',
+    time: '',
+  });
+
+  const [disabletItem, setDisabletItem] = useState<number | null>(null);
+
+  const [zeroing, setZeroing] = useState<boolean>(false);
```

Comment:
if you have several `useState` hooks in your component, makes sense to move them into single `useReducer`